barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.cpp

```diff
@@ -178,12 +178,12 @@ void ClientIVC::accumulate(ClientCircuit& circuit,
                            const std::shared_ptr<MegaVerificationKey>& precomputed_vk,
                            const bool mock_vk)
 {
-    // Construct merge proof for the present circuit and add to merge verification queue
-    MergeProof merge_proof = goblin.prove_merge(circuit);
-
     // Construct the proving key for circuit
     std::shared_ptr<DeciderProvingKey> proving_key = std::make_shared<DeciderProvingKey>(circuit, trace_settings);
 
+    // Construct merge proof for the present circuit
```

Comment:
There was no need for this `prove_merge` call to come after proving key construction - that was left over from when it used to add a recursive merge verification under the hood. It also isnt valid to add ops in that call because those ops were not being added when we constructed a VK from dummy witnesses in acir. They no longer need to be present anyway since the proving key constructor is adding some.